### PR TITLE
Profile page #5

### PR DESCRIPTION
**Pull request by eric-silva-61**
### What this PR does (required):
- Updated Profile to only display Contests that the user owns

### Screenshots / Videos (required):
p1
p2

### Any information needed to test this feature (required):
- Still not available due to integration into Profile Settings (will address this next)

### Any issues with the current functionality (optional):
- Please see comments

### CODE REVIEW

**Comment by eric-silva-61** [comment]
- [x] - I believe that this PR will also address #22 
- [x] - I had already made the updates from the comments in PR for #16 here prior to seeing them
- [x] - Need a default image to display when a contest has no submissions (addressed in #29 )
- [x] - I used MUI ImageList but have concerns about consistent image sizing without distortion.  Perhaps swap to a custom component with background image and cover.

**Comment by eric-silva-61** [comment]
It just occurred to me that I made a mistake in this PR.  Clicking on each tab causes a render cycle so there is no need to split the data up into open/closed contests and populate both tabs with data.  Old school thinking on my part.  Will fix it today.

File: server/routes/contest.js [hunk 1]
```diff
@@ -8,10 +8,12 @@ const {
   getContest,
   getContests,
   createSubmissionByContestId,
+  getUserContests,
 } = require('../controllers/contest');
 
 router.route('/').post(validateContest, createContest);
 router.route('/all').get(getContests);
 router.post('/:id/submission', protect, createSubmissionByContestId);
+router.route('/contests').get(protect, getUserContests);
```
Comment (rajivtitus) [inline server/routes/contest.js]: Maybe you can rename this route (for eg `user-contests`) for increased semantics?

File: client/src/pages/ProfileSetting/Profile/Profile.tsx [hunk 2]
```diff
@@ -23,40 +24,76 @@ const TabPanel = (props: TabPanelProps) => {
...
-  },
-];
-
 const Profile: FC = (): JSX.Element => {
   const classes = useStyles();
-  const { loggedInUser } = useAuth();
   const [value, setValue] = useState(0);
+  const [contests, setContests] = useState<ContestAPIData[]>([]);
+
+  useEffect(() => {
```
Comment (rajivtitus) [inline client/src/pages/ProfileSetting/Profile/Profile.tsx]: I know we discussed about this in the morning for a bit, having a look at the code I do notice that each time the Profile components mounts, all the contests for the user are being fetched and the sorting is being done below. Maybe consider using a context(existing/new) instead?

**Review by rajivtitus — COMMENTED** [review]
Looks good. 
Please see comment about implementation. Thanks

File: server/routes/contest.js (hunk 1 above)
Comment (eric-silva-61) [reply server/routes/contest.js]: Done

File: client/src/pages/ProfileSetting/Profile/Profile.tsx (hunk 2 above)
Comment (eric-silva-61) [reply client/src/pages/ProfileSetting/Profile/Profile.tsx]: I'm not sure what you mean.  Let's discuss briefly this morning in the stand up.

Comment (eric-silva-61) [reply client/src/pages/ProfileSetting/Profile/Profile.tsx]: I made a context for the contests, but still have to loop over the contests to get a count and a sample image.

**Review by rajivtitus — APPROVED** [review]
LGTM! Thanks for making requested changes

**Pull request merged**